src/components/TipInput.vue

```diff
@@ -99,7 +94,6 @@ export default {
     Modal,
     Dropdown,
     IconTip,
-    IconTipUser,
```

Comment:
agree is @onvisions will approve that

Reply:
actually, I don't have access to the latest designs of user profile page (can somebody share a link?), but as I see at #1040 they were already broken